src/Tooltip.tsx

```diff
@@ -99,13 +99,19 @@ const Tooltip: Tooltip = React.forwardRef<HTMLDivElement, TooltipProps>(
   ) => {
     bsPrefix = useBootstrapPrefix(bsPrefix, 'tooltip');
 
+    const [primaryPlacement] = placement?.split('-') || [];
+
     return (
       <div
         ref={ref}
         style={style}
         role="tooltip"
-        x-placement={placement}
-        className={classNames(className, bsPrefix, `bs-tooltip-${placement}`)}
+        x-placement={primaryPlacement}
+        className={classNames(
+          className,
+          bsPrefix,
+          `bs-tooltip-${primaryPlacement}`,
```

Comment:
**nitpick:** we shouldn't do `bs-tooltip-undefined` or whatever we'd get when `placement` is null, should we?

Reply:
this should be `primaryPlacement &&` as well